crates/autopilot/src/arguments.rs

```diff
@@ -359,6 +365,7 @@ impl std::fmt::Display for Arguments {
             max_settlement_transaction_wait
         )?;
         writeln!(f, "s3: {:?}", s3)?;
+        writeln!(f, "circuit breaker: {:?}", circuit_breaker)?;
```

Comment:
@sunce86 @MartinquaXD can we display the `authenticator_eoa` here? or should we hide it?

Reply:
The EOA is actually a private key so it should be hidden.

Reply:
In practice let's please KMS for this.